executor/metrics/pusher.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import (
+	mgostatsd "github.com/scullxbones/mgo-statsd"
+	"gopkg.in/mgo.v2"
+)
+
+// StatsdPush is a metrics pusher to Statsd
```

Comment:
comment on exported type StatsdPusher should be of the form "StatsdPusher ..." (with optional leading article)